Mage/src/main/java/mage/abilities/effects/common/DamageTargetEffect.java

```diff
@@ -67,15 +69,20 @@ public DamageTargetEffect(DynamicValue amount, boolean preventable) {
     }
 
     public DamageTargetEffect(DynamicValue amount, boolean preventable, String targetDescription) {
-        this(amount, preventable, targetDescription, false);
+        this(amount, preventable, targetDescription, false, DynamicValue.EffectPhrasing.EQUAL_TO);
     }
 
     public DamageTargetEffect(DynamicValue amount, boolean preventable, String targetDescription, boolean useOnlyTargetPointer) {
+        this(amount, preventable, targetDescription, useOnlyTargetPointer, DynamicValue.EffectPhrasing.EQUAL_TO);
+    }
+
+    public DamageTargetEffect(DynamicValue amount, boolean preventable, String targetDescription, boolean useOnlyTargetPointer, DynamicValue.EffectPhrasing phrasing) {
```

Comment:
can we please not put this param in the constructors for this class? It's already a mess in need of rework, so I'd prefer to just use the `withPhrasing` method